src/model/writer.js

```diff
@@ -867,7 +877,10 @@ export default class Writer {
 	 * The `options.usingOperation` parameter lets you change if the marker should be managed by operations or not. See
 	 * {@link module:engine/model/markercollection~Marker marker class description} to learn about the difference between
 	 * markers managed by operations and not-managed by operations. It is possible to change this option for an existing marker.
-	 * This is useful when a marker have been created earlier and then later, it needs to be added to the document history.
+	 *
+	 * The `options.affectData` parameter allows you to inform the engine, that this marker affects the data model.
```

Comment:
`affects the data model` It does not affects the data model. It affects data, which you get from editor. `affects the editor data` would be better.